src/Microsoft.DotNet.Build.Tasks/AddDependenciesToProjectJson.cs

```diff
@@ -22,10 +22,12 @@ public class AddDependenciesToProjectJson : Task
         // Additional Dependencies to add to the project.json. May Optionally contain a version.
         // Will Override dependencies present in the project if there is a conflict.
         // AdditionalDependencies required metadata:  Name, Version
-
         [Required]
         public ITaskItem[] AdditionalDependencies { get; set; }
 
+        // Framework section which the additional dependencies apply to.  Empty is the default dependencies section.
```

Comment:
See https://github.com/dotnet/corefx/blob/master/src/System.Globalization/tests/project.json which is an example where it has 2 target frameworks and a set of shared dependencies.